docs/ccv1_image_security_design.md

```diff
@@ -154,55 +154,33 @@ From its `layers` field, we can see the changes we stated above.
 
 ## Image signing
 
-The container image signing mechanism used by the image-rs of confidential containers V1 
-is consistent with the signing mechanism used in [containers/image](https://github.com/containers/image). 
-At present, the only supported cryptographic signature format in this mechanism is OpenPGP([RFC 4880](https://datatracker.ietf.org/doc/html/rfc4880#section-5.4)). 
+There are multiple image signing and verification protocols/solutions available now, 
```

Comment:
There are multiple image signing and verification protocols/solutions in the field, so an extensible modular architecture should be implemented to support them.

Reply:
Even if this doc doesn't list them all, is it worth collecting the solutions we're aware of and plan to recognise in an issue?

Reply:
@jodh-intel Done: #7